build/docker/intel-idxd-config-initcontainer.Dockerfile

```diff
@@ -14,12 +14,22 @@
 ## See the License for the specific language governing permissions and
 ## limitations under the License.
 ###
-FROM debian:unstable-slim
-COPY ./LICENSE /licenses/intel-device-plugins-for-kubernetes/LICENSE
+FROM debian:stable-slim as build
 RUN apt-get update && apt-get install -y --no-install-recommends accel-config jq && rm -rf /var/lib/apt/lists/\*
+RUN mkdir -p /idxd-init/scratch
+ARG LIBS="libjson libaccel libjq libtinfo libonig libacl libselinux libpcre2-8"
+ARG EXECS="bash env accel-config jq sed grep"
```

Comment:
maybe worth thinking if it made sense to bring in toybox here too?

Reply:
Replacing the debian tools with toybox or just having toybox as it's convenient?

Reply:
replacing bash, env, sed and grep from Debian with toybox